fullscreen.bs

```diff
@@ -505,9 +516,16 @@ properties apply to this pseudo-element either.
 <p>The <dfn id=css-pc-fullscreen><code>:fullscreen</code></dfn> pseudo-class must match any
 <a>element</a> that has its <a>fullscreen flag</a> set.
 
-<p class="note no-backref">This makes it different from the {{Document/fullscreenElement}} API,
+<p class="note no-backref">This makes it different from the
+{{DocumentOrShadowRoot/fullscreenElement}} API,
 which returns a <a>document</a>'s <a>fullscreen element</a>.
```

Comment:
In order to avoid repeating the details about "or shadow root" and retargeting here, mayb just say "This makes it different from the fullscreenElement API, which returns the topmost fullscreen element", without linking to the definition of "fullscreen element"? I fear that trying to make this technically correct will just make it harder to understand what the difference is.